app.test.js

```diff
@@ -46,4 +46,38 @@ describe('getCategories', () => {
             })
           });
       });
+})
+
+describe('getReviews', () => {
+  test("Responds with an object containing an array of review objects", () => {
+      return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({body}) => {
+          expect(body.reviews).toEqual(expect.any(Array));
```

Comment:
If you have a forEach in your test any assertions that you have an array are implied by the forEach (as it wouldn't work if it wasn't an array). Checking the length of an array is still valid though to assert it contains the expected number of elements.